spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -142,7 +145,19 @@ public ResponseEntity<String> manifest(@PathVariable("name") String name,
 	@RequestMapping(path = "/history/{name}", method = RequestMethod.GET)
 	@ResponseStatus(HttpStatus.OK)
 	public Collection<Release> history(@PathVariable("name") String releaseName) {
-		return this.streamService.history(releaseName);
+		ArgumentSanitizer sanitizer = new ArgumentSanitizer();
+		return this.streamService.history(releaseName)
+			.stream()
+			.map(release -> {
+				if (release.getConfigValues() != null && StringUtils.hasText(release.getConfigValues().getRaw())) {
+					try {
+						release.getConfigValues().setRaw(sanitizer.sanitizeJsonOrYamlString(release.getConfigValues().getRaw()));
+					} catch (JsonProcessingException e) {
+						logger.error("history:exception:" + e, e);
```

Comment:
Do you mind adding a descriptive message to the error, good chance its related to santiization.

Reply:
Ok